app/controllers/users/confirmations_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
```

Comment:
@apeksha2697  please remove this file from PR

Reply:
done